dist/fedora-infra/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM fedora:30
+
+# build: project sources
+ADD . /src
+WORKDIR /src
+
+# build: system utilities and libraries
+RUN dnf -y install g++ openssl-devel
+
+# build: pinned Rust stable toolchain (only use in case of temporary distro breakage)
+#ADD https://static.rust-lang.org/dist/rust-1.35.0-x86_64-unknown-linux-gnu.tar.gz rust.tar.gz
+#RUN tar -xf rust.tar.gz --strip 1 && ./install.sh
```

Comment:
This leaves a mess on the filesystem which ends right up in the production image without separate bulid and deployment containers.

I would suggest to just remove the comments, maybe replaced by a link to https://github.com/openshift/cincinnati/blob/48fb7d57eda9948984af3a1c258217137ea98dc0/dist/Dockerfile.build/Dockerfile#L13-L18.

Reply:
Ack, I dropped this altogether.